app/aldp/models.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.models import User
+
+from board.models import Message
+
+class DatestampedWithMessagesAndAuthor(models.Model):
+    author = models.ForeignKey(User, verbose_name='Modérateur', related_name='+')
+    # TODO: définir ceci en fonction de la date du starting_message
+    start_date = models.DateTimeField(verbose_name='Date de démarrage', blank=True, null=True,
+                                        db_index=True)
+    # TODO: définir ceci en fonction de la date du ending_message
+    end_date = models.DateTimeField(verbose_name='Date de fin', blank=True, null=True)
+    start_message = models.ForeignKey(Message, verbose_name='Message d\'annonce',
+                                         related_name='+', blank=True, null=True)
+    end_message = models.ForeignKey(Message, verbose_name='Message de cloture',
+                                       related_name='+', blank=True, null=True)
+
+    class Meta:
+        abstract = True
+        ordering = ['date_started']
+
+    def start(self):
+        assert self.start_date is None
```

Comment:
Je suis d'avis de silentieusement ignorer les cas ou ``self.start_date is not None``. 
En pratique, que ça soit une assertion (à moins que Django ne les désactive en prod) ou une autre exception, ça va faire une erreur 500.